view-samples/custom-hover-cards/README.md

```diff
@@ -0,0 +1,42 @@
+# Custom hover card on List view
+
+## Summary
+The following image shows a list with a custom hover added to a List view:
+
+On hover - Metadata on the column "Status" is made available via view formatting
+
+![screenshot of the sample](./Status_hover.png)
+
+## View requirements
+- This format can be applied to any column type though the example is based on - 
+
+Column Name|Type
+--------|---------
+Status  | Choice
```

Comment:
Same comment for this. Please add information about other columns referred in the JSON.

Reply:
Done